src/components/services/DockerSocketProxy.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <Generic :item="item">
+    <template #indicator>
+      <div class="notifs">
+        <strong v-if="running > 0" class="notif running" title="Running Containers">
+          {{ running }}
+        </strong>
+        <strong v-if="stopped > 0" class="notif stopped" title="Stopped Containers">
+          {{ stopped }}
+        </strong>
+        <strong v-if="errors > 0" class="notif errors" title="Error">
+          {{ errors }}
+        </strong>
+        <strong v-if="serverError" class="notif errors" title="Connection error to Docker Socket Proxy API">
+          ?
+        </strong>
+      </div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+export default {
+  name: "DockerSocketProxy",
+  components: {
+    Generic,
+  },
```

Comment:
Generic is now a registered globally, you don't have to import and declare it anymore to use it (you can also remove line 24)
```suggestion
```